mars/dataframe/expressions/datasource/dataframe.py

```diff
@@ -88,3 +96,16 @@ def tile(cls, op):
 def from_pandas(data, chunk_size=None, gpu=None, sparse=False):
     op = DataFrameDataSource(data=data, gpu=gpu, sparse=sparse)
     return op(data.shape, chunk_size=chunk_size)
+
+
+def from_tensor(tensor, chunk_size=None, gpu=None, sparse=False):
+    if tensor.ndim != 2:
+        raise NotImplementedError('Not support create DataFrame from {0} dims tensor', format(tensor.ndim))
+
+    # fetch the data from tensor firstly
+    data = tensor.execute()
```

Comment:
We need an op like `TensorToDataFrame` to implement it. `execute` should not be called in expressions, we should do the conversion in the execution of op `TensorToDataFrame`.